pkg/components/exporter/exporter.go

```diff
@@ -165,7 +165,6 @@ func NewDaemonSet(detail components.Detail, k *v1alpha1.Kepler) *appsv1.DaemonSe
 							{Name: "lib-modules", MountPath: "/lib/modules", ReadOnly: true},
 							{Name: "tracing", MountPath: "/sys", ReadOnly: true},
 							{Name: "kernel-src", MountPath: "/usr/src/kernels", ReadOnly: true},
-							{Name: "kernel-debug", MountPath: "/sys/kernel/debug"},
```

Comment:
@BGrasnick, could you please amend the commit message details to include why removing this is okay? e.g. 

> Since `/sys` is already mounted, we need not mount /sys/kernel/*